ChrisConquet/Huiswerk2

```diff
@@ -0,0 +1,54 @@
+
+using UnityEngine;
+
+[System.Serializable]
```

Comment:
Nette code! Eén vraag: waarom staat dit stuk voor de andere "using" statements?

Reply:
Ah, dat komt omdat ik dom was en per ongeluk twee scripts op dezelfde plek in github gedumpt heb...